### PR TITLE
Update the latest version of w16 image

### PR DESCRIPTION
**Pull request by vijayreddy1991**
https://github.com/Shippable/pm/issues/10926

image

Before updating the envs get the latest envs

### CODE REVIEW

File: install.ps1 [hunk 1]
```diff
@@ -23,7 +23,10 @@ Install-ChocoPackage python2
 Install-ChocoPackage nodejs-lts
 
 # Installing Git
-choco install -y git
+Install-ChocoPackage git
+$env:ChocolateyInstall = Convert-Path "$((Get-Command choco).path)\..\.."
```
Comment (rageshkrishna) [inline install.ps1]: Move lines 27 and 28 up to line 6. This should be done right after choc installation so everything else in the script can use Update-SessionEnvironment if they need to.

Comment (vijayreddy1991) [reply install.ps1]: ok

File: install.ps1 [hunk 2]
```diff
@@ -3,7 +3,8 @@ $ErrorActionPreference = "Stop"
 # Install choco
 Set-ExecutionPolicy Bypass -Scope Process -Force
 iex ((New-Object System.Net.WebClient).DownloadString('https://chocolatey.org/install.ps1'))
-
+$env:ChocolateyInstall = Convert-Path "$((Get-Command choco).path)\..\.."
+Import-Module "$env:ChocolateyInstall\helpers\chocolateyProfile.psm1"
 function Write-PackageInstall($package) {
```
Comment (rageshkrishna) [inline install.ps1]: Please have a blank line above this to maintain readability.

Comment (vijayreddy1991) [reply install.ps1]: ok

File: install.ps1 [hunk 3]
```diff
@@ -23,7 +24,8 @@ Install-ChocoPackage python2
 Install-ChocoPackage nodejs-lts
 
 # Installing Git
-choco install -y git
+Install-ChocoPackage git
+Update-SessionEnvironment
  $env:PATH = 'C:\Program Files\Git\cmd;{0}' -f $env:PATH ;
   [Environment]::SetEnvironmentVariable('PATH', $env:PATH, [EnvironmentVariableTarget]::Machine)
```
Comment (rageshkrishna) [inline install.ps1]: These two lines (29, 30) should be removed. It's no longer necessary after update-sessionenvironment is run.

Comment (vijayreddy1991) [reply install.ps1]: ok

**Pull request merged**